### PR TITLE
Create an Application

### PR DESCRIPTION
**Pull request by turnipdabeets**
- [x] link this document to bind and authenticate when merged

Click on `Visit Preview` to view the generated version: https://developer-docs-git-devx-1172-applications-beyondidentity.vercel.app/docs/v1/workflows/applications

### CODE REVIEW

File: docs/workflows/applications.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<div style={{position: 'relative', paddingBottom: 'calc(73% + 20px)', height: '0'}}>
+	<iframe src='https://demo.arcade.software/KmtiNsx4Z31MkogQdwST?embed&forceNoOpeningAnimation=true' frameBorder="0" style={{position: 'absolute', top: '0', left: '0', width: '100%', height: '100%'}}>
+	</iframe>
+</div>
+
+## Create Application by API
+
+An application can also be created from the [Beyond Identity API](https://developer.beyondidentity.com/api/v1). Before making any API calls you'll want to generate an API access token. Check out [API Tokens](./api-token) for help creating an access token.
+
+### Create an Application Resource Server
```
Comment (tomsik68) [inline docs/workflows/applications.md:78]: I'd suggest omitting the word Application here. We call the concept just "Resource Server" -- not "Application Resource Server".

```suggestion
### Create a Resource Server
```

File: docs/workflows/applications.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+-H "Content-Type: application/json" \
+-d "{\"authenticator_config\":{\"config\":{\"type\":\"$(CONFIGURATION_TYPE)\",\"invoke_url\":\"$(INVOKE_URL)\",\"trusted_origins\":[\"$(TRUSTED_ORIGIN)\"],\"invocation_type\":\"$(INVOCATION_TYPE)\"}}}"'
+title="/authenticator-configs"
+/>
+
+### Create an Application
+
+Once you have created an [Authenticator Config](#create-an-authenticator-config) and possibly a [Resouce Server](#create-an-application-resource-server), you can create an application.
+
+While the Resouce Server ID is optional, make sure to have the Authenticator Config ID for the applications request. Set the application's Display Name, Client Type, Token Configuration, and redirect URI.
```
Comment (tomsik68) [inline docs/workflows/applications.md:114]: The authenticator config ID is optional as well. Maybe it's not important for this guide, but I just wanted to call it out. You can create an application without authenticator config if you only want to use the client credentials OAuth flow.

Comment (ismailmustafa) [reply docs/workflows/applications.md:114]: What happens if you create an application without an authenticator config for the authorization code flow? Does it default to the web authenticator?

File: docs/workflows/applications.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: Applications
+sidebar_position: 5
+---
+
+import MultiLanguageCodeBlock from "../../src/components/MultiLanguageCodeBlock";
+
+# Application Overview
+
+An application is created within a realm to configure OAuth2/OIDC flows. It represents your application (e.g. AcmeCorp Android App, Todo-List iOS App, etc) and utilized its realm's directory, policy, events, and branding objects. The application contains client configuration, token configuration, and an [Authenticator Config](../platform-overview/authenticator-config.md).
```
Comment (ismailmustafa) [inline docs/workflows/applications.md:10]: Maybe something like this?

An application is an abstraction in the admin console that enables you to configure passkey binding and authentication in your mobile/web apps. An application contains client configuration, token configuration, and an [Authenticator Config](../platform-overview/authenticator-config.md). There are two ways to create an admin console application for your apps:
- One application per platform: In this scenario, there is one admin console application created per app on each platform your app supports (iOS, Android, Web, etc). This is easier to configure upfront and provides flexibility in regards to platform specific configurability. However, it is more difficult to configure the generation of passkey binding links across platforms.
- One application for all platforms: In this scenario, there is one admin console application created for your app across all platforms you support. This is more difficult to configure upfront as it required building a web routing layer to redirect passkey binding links appropriately, however it makes it easier to accomplish a more seamless experience when generating passkey binding links across platforms.

A realm contains multiple applications, and an application utilizes the realm's directory, policy, events, and branding objects. To see how an application fits in the wider Beyond Identity architecture, check out [Architecture](../platform-overview/architecture.md).

Comment (turnipdabeets) [reply docs/workflows/applications.md]: Yea this is great, updated!

File: docs/workflows/applications.md (hunk 2 above)
Comment (turnipdabeets) [reply docs/workflows/applications.md]: I spoke with Tomas and it sounds like the request will fail in that case. 

Updated the docs to reflect an optional authenticator config.

File: docs/workflows/applications.md (hunk 3 above)
Comment (turnipdabeets) [reply docs/workflows/applications.md]: I also added a prerequisite section for all the guide while I was linking this guide to bind and authenticate.

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**